tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -30,6 +30,7 @@ all: serial-targets \
 	kernel-gpu-snp-tarball \
 	kernel-gpu-tdx-experimental-tarball \
 	nydus-tarball \
+	ovmf-sev-tarball \
```

Comment:
I forced a cache miss so that ovmf-sev was built, and it was built fine.

There is only one concern. The ovmf build for SEV will be installed in `/opt/kata/share/ovmf/OVMF.fd`. Currently we don't build the vanilla ovmf (https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L271) but in case we do, the installations will overwrite each other. The tdvf flavor installs in `/opt/kata/share/tdvf`, perhaps you should consider installing the ovmf-flavor under `/opt/kata/share/ovmf-sev`.

If you agree to address that, it doesn't need to be on this PR. Actually it would be better to do on a spin off IMHO.

@fitzthum I haven't seen your [PR for SNP](https://github.com/kata-containers/kata-containers/pull/6700) yet but this comment might apply to it as well.

Anyway, this commit seems ready to go.

Reply:
So for SEV/SNP we need two OVMFs. One should be called AMDSEV.fd and will be used at first just for SEV (later it will be used for both). SNP uses the default OVMF, which will be built to OVMF.fd

Reply:
I found it. @UnmeshDeodhar there is needed to forward-port https://github.com/kata-containers/kata-containers/commit/eebd24bec4149b8ace35f5f17ee2172870182453 so that ovnf file is renamed to AMDSEV.fd

Reply:
Changes coming from https://github.com/kata-containers/kata-containers/commit/eebd24bec4149b8ace35f5f17ee2172870182453 were not added yet.